### ISSUE TITLE
Scrobbling doesn't work on non-Bandcamp hostnames

### ISSUE THREAD

**Issue by drguildo** [cgravolet/scroblr#125]
If a Bandcamp page uses a non-Bandcamp hostname (e.g. http://music.musicofthemoon.com/), then scrobbling doesn't seem to work.

`<meta property="fb:app_id" content="165661066782720">` seems like it might be a fairly simple and reliable way of checking for Bandcamp streaming pages.

**Comment by cgravolet** [comment]
It also means updating the permissions so scroblr initializes on every website that you visit, and last time I tried that it made a lot of users very upset.

**Comment by bonyboy** [comment]
@cgravolet would there a way to track it from bandcamp.com so that if you did a search for something like [ESPRIT 空想](http://esprit.100percentelectronica.com) that it would track the next url after you visit (or search from) the bandcamp url?

**Comment by bdukes** [comment]
@bonyboy, no, this is about the permissions you grant the extension when you install or upgrade it.  In order to have Scroblr know anything about the page on some arbitrary URL, the extension has to tell Chrome to give it access to pages on every domain, whereas right now it's restricted just to a list of music domains.  Chrome's extension model doesn't let the extension get permissions to a new domain in the midst of browsing, so it doesn't help to look for something on the current page, unfortunately.

**Comment by bonyboy** [comment]
@bdukes I see what you're saying. Thanks for taking the time to explain it to me.
